ha/core/system_health/const.py

```diff
@@ -92,4 +92,9 @@ class HEALTH_EVENT_ACTIONS(Enum):
     IGNORE = "ignore"
     UPDATE = "update"
     UPDATEPUBLISH = "updatepublish"
-    PUBLISH = "publish"
\ No newline at end of file
+    PUBLISH = "publish"
+
+# Specific_info attributes
+class SPECIFIC_INFO_ATTRIBUTES(Enum):
+    GENERATION_ID = 'generation_id'
```

Comment:
we are using generation_id directly all over HA.  Please create a ticket and keep it in backlog to use this constant instead of "generation_id" everywhere in HA

Reply:
Ticket created : https://jts.seagate.com/browse/CORTX-30834